how_to_build.md

```diff
@@ -44,6 +46,42 @@ After the build has completed, the build artifacts will be located in `artifacts
 
     `build.cmd spec pack`
 
+* Run a target without running its dependencies (might fail if the dependencies
+  haven't been previously built):
+
+    `build.cmd -s spec`
+
 * View the full list of options:
 
     `build.cmd -?`
+
+### Building only a subset of the supported target frameworks
+
+FakeItEasy targets multiple versions of .NET (.NET Framework 4.0 and 4.5, .NET
+Standard 1.6, 2.0 and 2.1), and the tests also run on multiple frameworks (.NET
+Framework 4.6.1 and several versions of .NET Core). A consequence is that a full
+build can take a significant time. When working on the code, you might want a

```

Comment:
"take a significant time" hits my ear funny. I'd've written "take a significant amount of time". Or perhaps "take significant time".

Reply:
Ah, thanks. Not being an native English speaker, it can sometimes be hard to tell what will sound funny to a native speaker...

Reply:
And it also shouldn't be netstandard2.1 for the netcore3.0 profile